yb-voyager/cmd/root.go

```diff
@@ -120,3 +135,37 @@ func validateExportDirFlag() {
 		exportDir = strings.TrimRight(exportDir, "/")
 	}
 }
+
+func lockExportDir() {
+	lockfileName, err := filepath.Abs(filepath.Join(exportDir, ".lockfile.lck"))
+	if err != nil {
+		fmt.Println("Unable to create an absolute path for the export-dir lockfile")
+		os.Exit(1)
+	}
+
+	lock, err = lockfile.New(lockfileName)
+	if err != nil {
+		fmt.Printf("Failed to init lock on export-dir. Unable to create lockfile %v. Reason: %v", lockfileName, err)
+		os.Exit(1)
+	}
+
+	err = lock.TryLock()
+	if err == nil {
+		fmt.Println("Got the lock on export directory")
+	} else if err == lockfile.ErrBusy {
+		fmt.Println("Unable to lock the export-dir. Another instance of yb-voyager is running in the export-dir.")
```

Comment:
No need to mention `Unable to lock the export-dir` -- that's an implementation detail irrelevant to the user.

Reply:
Done